examples/chip-tool/commands/common/CredentialIssuerCommands.h

```diff
@@ -62,6 +62,15 @@ class CredentialIssuerCommands
     virtual CHIP_ERROR SetupDeviceAttestation(chip::Controller::SetupParams & setupParams,
                                               const chip::Credentials::AttestationTrustStore * trustStore) = 0;
 
+    /**
+     * @brief
+     *   This function is used to set the path to Device Attestation revocation set JSON file.
+     *
+     * @param[in] path Path to the JSON file containing list of revoked DACs or PAIs.
```

Comment:
Apparently passing null is allowed?  This needs to be documented, along with what the semantics of passing null are.